### ISSUE TITLE
Doesn't work with latest web3.js

### ISSUE THREAD

**Issue by scriptin** [llSourcell/Your_First_Decentralized_Application#1]
Only works on web3@0.20.1, as mentioned in the [linked tutorial](https://medium.com/@mvmurthy/full-stack-hello-world-voting-ethereum-dapp-tutorial-part-1-40d2d0d807c2).

Fix: `npm install web3@0.20.1`

**Comment by Joe-Connolly** [comment]
Yes, several commands need to be updated or users will get problems like https://ethereum.stackexchange.com/questions/23842/deploy-contract-on-ethereumjs-testrpc

**Comment by letorruella** [comment]
Here are the commands updated to 1.0 on web3

Instead of:
`VotingContract = web3.eth.contract(abiDefinition)`  <= **OLD**   
 `VotingContract = new web3.eth.Contract(abiDefinition)`  <= **NEW**

`deployedContract = VotingContract.new(['Rama','Nick','Jose'],{data: byteCode, from: web3.eth.accounts[0], gas: 4700000})`    <= **OLD**


`deployedContract = VotingContract.deploy(['Rama','Nick','Jose'],{data: byteCode, from: web3.eth.accounts[0], gas: 4700000})`  <= **NEW**
